backend/src/API/mailAPI.ts

```diff
@@ -194,3 +195,39 @@ export const sendTECReminder = async (
   } pending this semester.\n${reminder}\nTo submit your TEC, please visit https://idol.cornelldti.org/forms/teamEventCredits.`;
   return emailMember(req, member, subject, text);
 };
+
+/**
+ * Send an email reminder to members who do not have a coffee chat blackout
+ * @param req - The request made when sending the email
+ * @param member - The member being sent the email
+ * @returns - The response body containing information of the member being sent the email
+ */
+export const sendCoffeeChatReminder = async (
+  req: Request,
+  member: IdolMember
+): Promise<AxiosResponse> => {
+  const subject = 'Coffee Chat Reminder';
+  const memberCoffeeChats: CoffeeChat[] = await Promise.all(
+    (await getCoffeeChatsByUser(member)).map(async (chat) => ({
+      ...chat
+    }))
+  );
+
+  let approvedCount = 0;
+  let pendingCount = 0;
+  memberCoffeeChats.forEach((chat) => {
+    if (chat.status === 'approved') {
+      approvedCount += 1;
+    }
+    if (chat.status === 'pending') {
+      pendingCount += 1;
+    }
+  });
+
+  const text = `Hey! You currently have ${approvedCount} coffee ${
+    approvedCount !== 1 ? 'chats' : 'chat'
+  } approved and ${pendingCount} coffee ${
```

Comment:
If we're only sending a reminder to newbies without bingos it might be good to customize this message so the policy is clear. If not, maybe adding a note like "If you're a newbie, remember you must submit at least one bingo"